libkbfs/key_manager_test.go

```diff
@@ -1706,22 +1712,12 @@ func testKeyManagerRekeyAddDeviceWithPrompt(t *testing.T, ver MetadataVer) {
 		t.Fatalf("Couldn't set rekey bit")
 	}
 
-	c := make(chan bool)
+	c := make(chan struct{}, 1)
 	// Use our other device as a standin for the paper key.
-	clta := &cryptoLocalTrapAny{config2Dev2.Crypto(), c, config2.Crypto()}
+	clta := &cryptoLocalTrapAny{config2Dev2.Crypto(), sync.Once{}, c, config2.Crypto()}
```

Comment:
You could also call `DisableUpdatesForTesting()` to pause updates, if that's what's causing the issue.